layouts/partials/footer.html

```diff
@@ -32,7 +32,7 @@
 
 
 
-								<i class="icon-envelope2"></i> <a href="https://groups.google.com/forum/#!forum/kiali-users">Kiali Users group</a> <span class="middot">&middot;</span> <i class="icon-envelope2"></i> <a href="https://groups.google.com/forum/#!forum/kiali-dev">Kiali Developers group
+								<a href="https://groups.google.com/forum/#!forum/kiali-users"><i class="icon-envelope2"></i> Kiali Users group</a> <span class="middot">&middot;</span> <a href="https://groups.google.com/forum/#!forum/kiali-dev"><i class="icon-envelope2"></i> Kiali Developers group
 
```

Comment:
Missing the close ```</a>``` at the end of the line, it seems that was missed before this change :( Can you add it?
The problem with this is that if we add the ```</a>``` we'll see the license in black because it was using the same css by the ```<a>```

![after](https://user-images.githubusercontent.com/3019213/50274967-9a9d6300-043e-11e9-9be9-5ec819d50ef7.png)

So my proposal is add this part to the same link because is related with the license
```html
<div class="widget clearfix">
  <p>
    <a href="https://www.apache.org/licenses/LICENSE-2.0.txt">
      <strong>License:</strong> 
      Kiali is open source and released under Apache License v2
    </a>.
  </p>
</div>
```